### PR TITLE
Reworking Integration

### PR DESCRIPTION
**Pull request by smark88**

### CODE REVIEW

**Comment by smark88** [comment]
Waiting for merge of https://github.com/broadinstitute/datarepo-actions/pulls

**Review by snf2ye — APPROVED** [review]
LGTM - just make sure to switch back to the action version.

File: .github/workflows/dev-image-update.yaml [hunk 1]
```diff
@@ -3,6 +3,15 @@ on:
   push:
     branches:
       - develop
+    paths:
+      - '!*'
+      - 'src/**'
+      - 'package.json'
+      - 'cypress.json'
+      - 'cypress/**'
```
Comment (fboulnois) [inline .github/workflows/dev-image-update.yaml:11]:
```suggestion
      - 'cypress/**'
      - 'public/**'
```

File: .github/workflows/test-e2e.yml [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
 on:
   pull_request:
     branches:
       - develop
+    paths:
+      - '!*'
+      - 'src/**'
+      - 'package.json'
+      - 'cypress.json'
+      - 'cypress/**'
```
Comment (fboulnois) [inline .github/workflows/test-e2e.yml:17]:
```suggestion
      - 'cypress/**'
      - 'public/**'
```

**Review by fboulnois — APPROVED** [review]
@myessail should get the final say on which paths trigger a version bump

**Review by myessail — APPROVED** [review]

**Review by nmalfroy — APPROVED** [review]
lgtm.  I'm wondering about more paths that we may want to include but we can do that as needed if it comes up

**Pull request merged**